Danielle/updates

dnkem opened the pull request:
# Making Updates to the Overall Project
- replace this with a short description of what the pr is for

# Changes
- Removed the borders and shadows between the sections
- Removed the extra black space on the right side

# Additional Info
- There's issues with the images of the stars and some show shadows or are cut off

# Screenshots (if necessary)
<img width="1506" alt="Screenshot 2024-11-27 at 12 58 58 AM" src="https://github.com/user-attachments/assets/bf1359e1-e3a9-4ab4-91c1-8893cc2468e6">
<img width="1512" alt="Screenshot 2024-11-27 at 12 59 18 AM" src="https://github.com/user-attachments/assets/d2504e87-04cd-43dc-8248-a4c73fd83cdc">

Original | Updated
:------------------------:|:------------------------:
original screenshot | updated screenshot

Kanik-a committed:
Desktop and Mobile Navbar

Kanik-a committed:
Updated navbar

Kanik-a committed:
Merge branch 'main' into Desktop-navbar

Kanik-a committed:
Navbar-Mobile and Laptop

ney927 committed:
commit to get deploy preview

ney927 committed:
Merge branch 'main' into Desktop-navbar

ney927 committed:
media queries work but black bar exists on side

dnkem committed:
fixed the individual divs and the lines in between

- the about-us image with the stars is messed up and has a shadow that's edited in.

- the 'everything' div isn't widening to 100%

- star background image is cropped too small in the milestones

- the navbar is not widening to 100% width

dnkem committed:
Fixed black gaps on the side of the page

There was an issue with the homepage container and the "width: 100%;" over corrected it but it's now fixed by removing a line of code from there and the other widths from all other files.

ney927 committed:
Merge branch 'Desktop-navbar' into danielle/updates

ney927 commented:
Amazing! 🤩

ney927 committed:
integrated updated navbar

ney927 commented:
## Current Issues:

### 1. images are appearing behind the subheadings when an overlap occurs:

- ![image](https://github.com/user-attachments/assets/4ef99bb2-377a-464b-825c-6e444963b46f)
- ![image](https://github.com/user-attachments/assets/2527524f-c821-4018-acc4-1641982f7818)

### 2. black bar on the right side persists for screens smaller than ~430px:
 - ![image](https://github.com/user-attachments/assets/fa874c0e-8bea-44aa-8d82-9fc4ef3e63a5)
    
### 3. media queries for statistics for smaller screen
 - I would recommend that the circles become more stacked so that they are not too small and unreadable
 - ![image](https://github.com/user-attachments/assets/ec4d90c9-7a25-441d-93e9-0c65fbb9858e)

### 4. navbar logo is out of place for screens larger than 1000px
- ![image](https://github.com/user-attachments/assets/36d46e53-0572-40e0-bfd0-3790660610ae)

### 5. desktop navbar needs updated hover and dropdown stylings. also the shadow on the bottom
- current design:
    - ![image](https://github.com/user-attachments/assets/24412e7e-d2c5-4419-8613-8d89b8bc8f60)
- figma design:
    - ![image](https://github.com/user-attachments/assets/5a346197-e230-4a9e-b388-124a62441057)
    - [see design here](https://www.figma.com/design/cUxhBXDDa5WpdDQryf8FNo/Technolgap-Desktop)

### 6. mobile navbar needs updated hover and dropdown stylings. also add the fade on the rest of the page
- current design:
    - ![image](https://github.com/user-attachments/assets/6ca8e6ae-3f2e-41de-968e-59f3a3f68087)
- figma design: 
    - ![image](https://github.com/user-attachments/assets/6aaa2d83-9874-4903-be30-b8623aa6c763)
    - [see design here](https://www.figma.com/design/ks0uMqVJMkkQmAcdxVysl2/Technolgap-Mobile)

ney927 committed:
updates

scroll to top button
responsive milestones
responsive core values
responsive subheading
updated navbar links
